### ISSUE TITLE
How to upload folder to Glacier using multi part

### ISSUE THREAD

**Issue by kenrai123** [MoriTanosuke/glacieruploader#46]
Hi team,

How to upload folder to Glacier using multi part, please help me.

Thanks team.

**Comment by MoriTanosuke** [comment]
Hey @kenrai123, I'll have a look at this in the evening. I'm not sure if uploading a folder is possible to the moment. A quick workaround would be creating a regular archive (tarball or gzip) of the folder and upload that instead. 

BTW, there is no team here. 😉

**Comment by MoriTanosuke** [comment]
I checked the sourcecode and there is now way to upload a folder. If you want to upload all files from a folder I suggest creating a script and run glacieruploader multiple times.

The AWS sdk only handles single archives (any file like pictures, documents, zip files...) and glacieruploader only works on those single archives. When I started this project I always thought about the usage like one of the UNIX tools: small programs that you put together using pipes and scripts.

Here's what I would do:

* use a `find` command to get all files from a folder
* use the `-exec` switch of `find` to run glacieruploader for each file

Hope this helps.
